docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrGit_Source_Code.xml

```diff
@@ -0,0 +1,610 @@
+  directly from Git repositories. The repositories (and optional
+  branches/users) are configured using environment variables on the server.
+  Using the Git hooks you can submit a query from a repository branch, and the
+  ECLCC Server will pull the source code from a Git repository and compile it.
+  This allows you to deploy a specific version of a query without needing to
+  perform any work on the client.</para>
+
+  <sect2 id="TheGitImrpovements">
+    <title>The Git Improvements</title>
+
+    <para>Starting with version 8.4, the platform code for Git support
+    significantly improved. Some of these improvements have been backported to
+    older support releases such, as 7.12. However, You still need to update to
+    a recent point release to ensure you get any of these improvements. While
+    the later releases such as 8.6 will include all of these
+    improvements.</para>
+
+    <sect3 id="TheSPeedIMprovements">
+      <title>Speed Improvements</title>
+
+      <para>The platform code has been upgraded for significant improvements
+      to the speed. For instance, on very large queries the compile time has
+      been reduced by literally dozens of minutes. In time sensitive
+      environments where there are SLA requirements this can be very
+      significant.</para>
+    </sect3>
+
+    <sect3 id="GitResourcesAndManifests">
+      <title>Git Resources and Manifests</title>
+
+      <para>Previous to these Git improvements, if Git resources and manifests
+      were contained within the source files, they could be associated with a
+      query. With these recent improvements to the HPCC Systems platform now
+      supports manifests and resources when compiling using the Githook
+      mechanism.</para>
+    </sect3>
+
+    <sect3 id="GitLFs_Support">
+      <title>Git-lfs Support</title>
+
+      <para>Git-lfs is an extension to Git that improves support for large
+      files and is supported by both GitHub and GitLab. This extension is
+      particularly useful for large resources. For example, if you have java
+      packages included as part of the manifest.</para>
+    </sect3>
+
+    <sect3 id="MultipleRepositorySupprt">
+      <title>Multiple Repository Support</title>
+
+      <para>The HPCC Systems platform code includes support for using multiple
+      Git repositories. Previous to these latest improvements you could only
+      compile from a single repository.</para>
+
+      <para>Using a single repository had some drawbacks. You could have
+      multiple copies of the same source code not really being in sync. You
+      could also have problems with incompatible changes, dependencies, or
+      clashes if there are modules with the same name. Using multiple
+      repositories solves these issues. Queries compiled from multiple
+      repositories, avoid problems with clashing symbols. It prevents clashes
+      and ensures that the dependencies between repositories are
+      versioned.</para>
```

Comment:
It may be simpler to delete this whole paragraph and add something to the following paragraph about it enabling development by several independent teams.